gradio/routes.py

```diff
@@ -1764,8 +1767,101 @@ async def analytics_dashboard(key: str):
             else:
                 raise HTTPException(status_code=403, detail="Invalid key.")
 
-        app.include_router(router)
+        @app.post("/process_recording", dependencies=[Depends(login_check)])
```

Comment:
This should be `router.post()` so it prefixes `/gradio_api/` in front of the URL so that we don't have conflicts with any custom pages that the user creates. We'll need to make the corresponding changes in the frontend as well.
```suggestion
        @router.post("/process_recording", dependencies=[Depends(login_check)])
```